stories/Tabs.stories.js

```diff
@@ -32,8 +32,12 @@ const Template = (args) => ({
         <div
             :class="args.jsDisabled ? 'no-js' : ''"
         >
-            <VsTabs v-bind="args">
-                <VsTabItem title="Getting Here">
+            <VsTabs 
```

Comment:
Very minor, but could this either be all on one line as it was before, or have the > on its own line below the props

<VsTabs v-bind="args">

<VsTabs
    v-bind="args"
>